app/frontend/javascript/multi-stamp-tubes/components/MultiStampTubes.vue

```diff
@@ -142,6 +146,12 @@ export default {
     // Should tubes be required to be in passed state
     // Also referenced as require-tube-passed and require_tube_passed
     requireTubePassed: { type: String, required: true },
+
+    // A acceptable list of purpose names that can be scanned
+    // If left empty all purposes are acceptable
+    // Also referenced as data-acceptable-purposes and data_acceptable_purposes
+    // See computed method acceptablePurposesArray for conversion to array.
+    acceptablePurposes: { type: String, required: false, default: '[]' },
```

Comment:
My understanding was it has to be a string because of the way the props are passed/injected into the component via the html tags. https://github.com/sanger/limber/pull/1881/files#diff-da315a25389aaf6400141ceef6736ae316fc9463d9e028990c124f8714d172d3.